applications/google_rise_2016.md

```diff
@@ -0,0 +1,459 @@
+volunteers), or hire teachers (they teach using our pre-written workshops). This
+model makes the cost of starting a new club virtually $0 from our end and we
+think it'll let us scale where others haven't.
+
+Our content is entirely open source and designed for community contribution.
+Though we're still in the early stages of driving community contributions, half
+of our curriculum has been built by the community. Having our content open
+source and accessible to contribution has the advantage of being self-correcting
+and continuously improved by club leaders and members.
+
+In terms of pedagogy, our curriculum isn't built to teach people how to program:
+it's built to help our club members become self-taught programmers. Rather than
+starting by teaching the fundamentals, we throw our members in at the deep end
+by having them build real projects and use Google when they get stuck. By the
+end of the first club meeting, everyone has built their first website and
+deployed it using git, GitHub, and GitHub Pages.
+
+**Has this organization applied for, or received, funding from Google
+previously? If yes, please describe.**
+
+No
+
+### Primary Contact
+
+Please fill out the information below for the primary contact for this grant.
+The RISE Awards team would reach out to this primary contact with questions, if
+needed.
+
+**Full name of primary point of contact for this grant (first and last name)**
+
+Zachary Latta
+
+**Title of primary contact**
+
+Executive Director
+
+**Primary contact e-mail address**
+
+zach@hackclub.com
+
+**Primary contact telephone number (including international country code)**
+
+[REDACTED]
+
+### About the Project
+
+Please provide us with details about the specific computer science education
+project that you would like the RISE Awards to fund.
+
+**Project name**
+
+Hack Club (the whole organization)
+
+**Project description (one sentence):**
+
+Hack Club helps high schoolers start coding clubs through providing prospective
+club leaders with open-source materials and a community of other leaders around
+the globe.
+
+**Detailed project description (include all activities):**
+
+We would like RISE to fund the continued expansion of Hack Club during 2016.
+Specifically, we're looking to bring someone on full-time to lead the
+development of our curriculum and would like RISE to fund their salary during
+their first 4-6 months.
+
+To expand on what Hack Club does:
+
+We provide everything high schoolers need to start amazing coding clubs: meeting
+guidelines, curriculum, template slideshows, and a supportive community of
+leaders around the globe.
+
+Club meetings are run in pairs and are like mini-hackathons: the first meeting
+in the pair kicks off with a workshop and the second culminates in demos.
+Members are shipping something in every meeting, whether from a workshop or from
+what they built for the mini-hackathon. By the end of the first meeting,
```

Comment:
Wait, what's the difference (for my own understanding)? "From a workshop" vs "from a mini-hackathon"? Isn't the mini-hackathon an extension of the workshop? Also, is pointing out the distinction important?